packages/component-model-manuscript/entities/manuscript/index.js

```diff
@@ -143,6 +143,22 @@ class Manuscript extends BaseModel {
     return manuscript
   }
 
+  static async findByIdAndAuthor(id, user, related = '[teams, files]') {
```

Comment:
Suggestion: Maybe we could use find in the policy and assert `manuscript.created_by === user`. This piece of work (once added to the resolvers) should make the querying by `id` and `created_by` unnecessary as we'll be able to do a `findById` instead.

Reply:
I was working on the assumption that what is currently `find` (but should be called `findByIdAndAuthor`) will change to what should be called `findById`, so I created the `findByIdAndAuthor` function based on the current `find` function so we don't have to fetch and process a load of manuscripts for every request, we can just check if this `userId` matches up to the `Manuscript` they're requesting, without actually having to process any `Manuscript` objects

Reply:
I'm thinking that the `Manuscript` would only need one find function that takes an id. There's no use-case for querying by id and userId right now other than for authorisation which this bit of middleware should be in control of in future. Better to get this policy working with the intended `find` then add the extra function to the model?

Reply:
Yeah, that makes a lot more sense, I'll do that